cdap-ui/app/directives/fileselect/fileselect.html

```diff
@@ -1,5 +1,10 @@
 
-<button class="btn btn-default" ng-class="buttonClass" ng-disabled="buttonDisabled">
+<button class="btn btn-default" ng-class="buttonClass" ng-disabled="buttonDisabled" ng-if="!isDropdown">
   <span class="fa fa-plus"> </span>
   <span> {{buttonLabel}} </span>
 </button>
+
+<a role="menuitem" ng-if="isDropdown">
```

Comment:
Same here too. wrapping ```{{buttonLabel}}``` in a span would be better. But I like this simplicity! Nice!